pom.xml

```diff
@@ -236,7 +236,10 @@
                         <packageLineRate>85</packageLineRate>
                         <packageBranchRate>85</packageBranchRate>
                     </check>
-                    <format>xml</format>
+                    <formats>
+                        <format>xml</format>
+                        <format>html</format>
+                    </formats>
                     <maxmem>256m</maxmem>
```

Comment:
Jayaram, could you please remove this change from pom.xml.

Reply:
Removed . Thanks @arao6